research/learning_to_remember_rare_events/train.py

```diff
@@ -197,7 +197,8 @@ def run(self):
 
         # validation
         correct = []
-        correct_by_shot = dict((k, []) for k in xrange(self.episode_width + 1))
+        num_shots = episode_length // episode_width
```

Comment:
Why is this being changed? Looks wrong, like regressing back to a previous version.